mon-pix/app/controllers/campaigns/campaign-landing-page.js

```diff
@@ -1,9 +1,43 @@
 import { action } from '@ember/object';
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
 
 export default class CampaignLandingPageController extends Controller {
+  @service currentDomain;
+  @service intl;
+  @service locale;
   @service router;
+  @service session;
+
+  @tracked selectedLanguage = this.intl.primaryLocale;
+
+  get canDisplayLanguageSwitcher() {
```

Comment:
Le nommage `canDisplayLanguageSwitcher` n'est pas assez impératif, voici ci-dessous une proposition même si elle ne me parait pas optimale, mais je ne trouve rien de mieux pour l'instant et c'est un nommage que l'on trouve beaucoup dans le code :

```suggestion
  get shouldDisplayLanguageSwitcher() {
```